ctapipe/reco/HillasReconstructor.py

```diff
@@ -218,30 +220,57 @@ def initialize_hillas_planes(
         k = next(iter(telescopes_pointings))
         horizon_frame = telescopes_pointings[k].frame
         for tel_id, moments in hillas_dict.items():
-            # we just need any point on the main shower axis a bit away from the cog
-            p2_x = moments.x + 0.1 * u.m * np.cos(moments.psi)
-            p2_y = moments.y + 0.1 * u.m * np.sin(moments.psi)
-            focal_length = subarray.tel[tel_id].optics.equivalent_focal_length
 
             pointing = SkyCoord(
                 alt=telescopes_pointings[tel_id].alt,
                 az=telescopes_pointings[tel_id].az,
                 frame=horizon_frame,
             )
 
-            camera_frame = CameraFrame(
-                focal_length=focal_length, telescope_pointing=pointing
-            )
+            focal_length = subarray.tel[tel_id].optics.equivalent_focal_length
 
-            cog_coord = SkyCoord(x=moments.x, y=moments.y, frame=camera_frame,)
-            cog_coord = cog_coord.transform_to(horizon_frame)
+            if moments.x.unit == u.Unit("m"):  # Image parameters are in CameraFrame
```

Comment:
`u.m`

Reply:
More readable and  faster as it uses a constant instead of creating a new unit by parsing a string

Reply:
DONE